tests/phpunit/CRM/Pledge/BAO/PledgePaymentTest.php

```diff
@@ -134,7 +134,8 @@ public function testRetrieveKnownPledgeID() {
   public function testDeletePledgePaymentsNormal() {
```

Comment:
@eileenmcnaughton @monishdeb this test seems silly because `CRM_Pledge_BAO_PledgePayment::deletePayments` will always return `TRUE` no matter how many pledge payments are deleted